src/bp/core/converse/converse-service.ts

```diff
@@ -116,10 +116,16 @@ export class ConverseService {
     })
   }
 
-  private async _createDonePromise(userKey: string) {
+  private async _createDonePromise(botId, userKey: string) {
     return new Promise((resolve, reject) => {
       converseApiEvents.once(`done.${userKey}`, async event => {
-        await Promise.delay(250)
+        let bufferDelay = _.get(await this.configProvider.getBotConfig(botId), 'converse.bufferDelayMs')
+
+        if (!bufferDelay) {
+          bufferDelay = _.get(await this.configProvider.getBotpressConfig(), 'converse.bufferDelayMs', 250)
+        }
```

Comment:
I think you should also add the config in `src/bp/core/config/botpress.config.ts` and `src/bp/core/config/bot.config.ts`

Reply:
Yes i forgot about that, thanks